libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKConstants.h

```diff
@@ -58,9 +58,9 @@ __builtin_unreachable(); \
 }
 
 #ifdef __clang__
-#define SFSDK_DEPRECATED(version, msg) __attribute__((deprecated("Deprecated in Salesforce Mobile SDK " #version ". " msg)))
+#define SFSDK_DEPRECATED(dep_version, rem_version, msg) __attribute__((deprecated("Deprecated in Salesforce Mobile SDK " #dep_version " and will be removed in Salesforce Mobile SDK " #rem_version ". " msg)))
```

Comment:
My only thought about this is that, if you don't have a clear idea of when you'll remove a deprecated item, you're forced into making that decision anyway.